configs/schema/configJsonSchema.js

```diff
@@ -243,19 +234,23 @@ module.exports = {
                 rateLimiter: {
                   type: 'object',
                   properties: {
-                    enable: {
-                      type: 'boolean',
+                    maxTokens: {
+                      type: 'integer',
+                      minimum: 0,
                     },
-                    rate: {
+                    tokensPerFill: {
                       type: 'integer',
                       minimum: 0,
                     },
-                    burst: {
+                    fillInterval: {
+                      type: 'string',
+                    },
```

Comment:
Let's add `pattern: '^[0-9]+(.[0-9]+)?(m|s|h)$',`

Reply:
Add also `ms` resolution

Reply:
I guess the question mark is unnecessary there.